content/general/concepts/software-development-life-cycle/terms/rapid-application-development/rapid-application-development.md

```diff
@@ -0,0 +1,42 @@
+ ---
+Title: 'Rapid Application Development (RAD)'
+Description: 'Rapid Application Development (RAD) is a software development methodology that emphasizes quick prototyping and iterative development to meet the requirements of a project.'  
+
+Subjects:
+  'Mobile Development'
+  'Web Development'  
+Tags:
+  'Documentation'
+  'Best Practices'
+  'Development'  
+CatalogContent:
+  'introduction-to-open-source'
+  'paths/computer-science'
+---
+**Rapid Application Development (RAD)** is a software development methodology that emphasizes quick prototyping and iterative development to meet the requirements of a project. The primary goal of RAD is to reduce development time and costs by quickly delivering a working prototype to the client for feedback and iteration.
+
+## Origins 
+
+RAD was developed in the 1980s by James Martin as a response to the Waterfall model, which was the dominant methodology at the time. Martin wanted to create a more efficient approach to software development that could handle fast-changing requirements and enable quick delivery of applications. The RAD approach was based on iterative development, where the emphasis was on building a prototype quickly and then refining it based on feedback from stakeholders.
+
```

Comment:
I think it would be helpful to go over the phases for RAD here

Reply:
Please address this @esand68 Refer to phases in the entry: https://www.codecademy.com/resources/docs/general/software-development-life-cycle